### ISSUE TITLE
Question: attempt to take screenshot on successful scenario steps, expected?

### ISSUE THREAD

**Issue by walterdolce** [elvetemedve/behat-screenshot#27]
Hi guys, I was wondering whether the extension behaviour explained below is expected or not.

I have a successful `Scenario Outline` step that gets executed, but it looks like an attempt to take a screenshot is made:
```
| zh-Hant-CN   |
      Screenshots are not supported by Behat\Mink\Driver\GoutteDriver
```

The extension configuration is as follows:
```
  extensions:
    Bex\Behat\ScreenshotExtension:
      image_drivers:
        local:
          screenshot_directory: ./
      screenshot_taking_mode: failed_scenarios
```

Shouldn't the message appear only when a failed scenario occurs?

**Comment by walterdolce** [comment]
Hold on. I was using version `1.2.2`. Updating to `1.2.4` I can see there's no message shown.
Closing. Thanks!

**Comment by tkotosz** [comment]
yep this was fixed in `1.2.3` :)
